test/test_indexing.py

```diff
@@ -343,10 +326,10 @@ def test_empty_tuple_index(self):
     def test_empty_fancy_index(self):
         # Empty list index creates an empty array
         a = tensor([1, 2, 3])
-        self.assertEqual(a[[]], Variable(torch.Tensor()))
+        self.assertEqual(a[[]], torch.Tensor())
 
         b = tensor([]).long()
-        self.assertEqual(a[[]], Variable(torch.LongTensor()))
+        self.assertEqual(a[[]], torch.LongTensor())
```

Comment:
here too; you might have to specify a dtype because it defaults to the default type.